client/my-sites/upgrades/index.js

```diff
@@ -222,38 +212,19 @@ module.exports = function() {
 		);
 	}
 
-	if ( config.isEnabled( 'upgrades/domain-management/list' ) ) {
-		page(
-			'/domains',
-			controller.siteSelection,
-			controller.sites
-		);
-
-		page(
-			'/domains/:site',
-			controller.siteSelection,
-			controller.navigation,
-			controller.jetPackWarning,
-			domainManagementController.domainManagementIndex
-		);
-	} else {
-		page(
-			'/domains',
-			adTracking.retarget,
-			controller.siteSelection,
-			upgradesController.domainsAddHeader,
-			controller.jetPackWarning,
-			controller.sites
-		);
+	page(
+		'/domains',
+		controller.siteSelection,
+		controller.sites
+	);
 
-		page( '/domains/:domain',
-			adTracking.retarget,
-			controller.siteSelection,
-			controller.navigation,
-			controller.jetPackWarning,
-			upgradesController.domainSearchIndex
```

Comment:
It looks like with this change `upgradesController.domainSearchIndex` becomes unused anywhere in our code and can be removed :raised_hands: